wildlifelicensing/apps/customer_management/views/customer.py

```diff
@@ -144,6 +147,81 @@ def _build_data(self):
 
         return data
 
+    #########################
+    # Applications
+    #########################
+    @property
+    def applications_table_options(self):
+        result = super(CustomerLookupView, self).applications_table_options
+        result.update({
+            'pageLength': 10,
+        })
+        return result
+
+    @property
+    def applications_data_url(self):
+        return reverse('wl_customer_management:data_applications', args=self.args)
+
+    @property
+    def applications_filters(self):
+        # no filters
+        return {}
+
+    @property
+    def get_applications_session_data(self):
+        # no session
+        return {}
+
+    #########################
+    # Licences
+    #########################
+    @property
+    def licences_table_options(self):
+        result = super(CustomerLookupView, self).licences_table_options
+        result.update({
+            'pageLength': 10,
+        })
+        return result
+
+    @property
+    def licences_data_url(self):
+        return reverse('wl_customer_management:data_licences', args=self.args)
+
+    @property
+    def licences_filters(self):
+        # no filters
+        return {}
+
+    @property
+    def get_licences_session_data(self):
+        # no session
+        return {}
+
+    #########################
+    # Returns
+    #########################
+    @property
+    def returns_table_options(self):
+        result = super(CustomerLookupView, self).returns_table_options
+        result.update({
+            'pageLength': 10,
+        })
+        return result
+
+    @property
+    def returns_data_url(self):
+        return reverse('wl_customer_management:data_returns', args=self.args)
+
+    @property
+    def returns_filters(self):
+        # no filters
+        return {}
+
+    @property
+    def get_returns_session_data(self):
+        # no session
+        return {}
+
     def get(self, request, *args, **kwargs):
```

Comment:
This is the base class that all Tables<...> will extend and override all the `@properties`